<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect_session</class>
 <widget class="QWidget" name="connect_session">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="ip_lbl">
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="ip_txt"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="port_lbl">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="port_txt"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="role_selector">
      <item>
       <property name="text">
        <string>Server</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Client</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
